test/dash/mpd_utils_unit.js

```diff
@@ -452,11 +452,9 @@ describe('MpdUtils', () => {
                       ' />');
       }
       xmlLines.push('</SegmentTimeline>');
-      const parser = new DOMParser();
-      const xml =
-          parser.parseFromString(xmlLines.join('\n'), 'application/xml');
-      const segmentTimeline = xml.documentElement;
-      console.assert(segmentTimeline);
+      const segmentTimeline = /** @type {shaka.extern.xml.Node} */ (
+        shaka.util.TXml.parseXmlString(xmlLines.join('\n'),
```

Comment:
nit: add two spaces in the indentation

Reply:
The linter reverts to this spacing unfortunately.